cart.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Cart Page</title>
+</head>
+<body>
+    
+    <nav class = "navigation flex gap-xl align-center full-width">
+        <div class="nav-brand">Water-Commerce</div>
+        <div class = "input-wrapper flex grow-1 flex-center" >
+           <input class = "input-field" placeholder="Please Enter Text">
+        </div>
+        <div class = "flex gap-xl flex-center grow-1" >
+           <button class="btn btn-secondary">Login</button>
+           <i class="material-icons" >favorite_border</i>
+           <i class="material-icons" >shopping_cart</i>
+        </div>
+    </nav>
+
+    <div class="text-m center-text m5-top bold">My Cart</div>
+
+    <div class = "container center-block m5-top flex space-between flex-center">
+        
+        <div class="card-container" style = "width:40%">
+            <div class="horizontal-badge flex no-wrap">
+                <div class="img-container">
+                    <img src="https://images.unsplash.com/photo-1643313947128-849e4da4fd5b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=2142&q=80"/>
+                </div>
+                <div class="card-header horizontal-card-header grow-1">
+                    <h4 class = "primary-text m2-top">Premium Jacket</h4>
+                    <p class="card-subtitle m1 secondary-text m2-top">$200</p>
+                    <label for="quantity">Quantity:</label>
+                    <input type="number" id="quantity" name="quantity">
```

Comment:
here you can use the min, max property of the input tag to restrict the user from going -ve in quantity

Reply:
Understood, thanks